powerauth-data-adapter/src/main/java/io/getlime/security/powerauth/app/dataadapter/impl/validation/AuthenticationRequestValidator.java

```diff
@@ -76,7 +76,7 @@ public void validate(@Nullable Object o, @NonNull Errors errors) {
             }
 
             ValidationUtils.rejectIfEmptyOrWhitespace(errors, "requestObject.organizationId", "login.organizationId.empty");
-            if (username!=null && organizationId.length() > 256) {
+            if (organizationId!=null && organizationId.length() > 256) {
```

Comment:
nitpick: missing spaces in `organizationId!=null`

Reply:
Fixed